pkg/github/repositories.go

```diff
@@ -796,3 +796,147 @@ func PushFiles(getClient GetClientFn, t translations.TranslationHelperFunc) (too
 			return mcp.NewToolResultText(string(r)), nil
 		}
 }
+
+// ListTags creates a tool to list tags in a GitHub repository.
+func ListTags(getClient GetClientFn, t translations.TranslationHelperFunc) (tool mcp.Tool, handler server.ToolHandlerFunc) {
+	return mcp.NewTool("list_tags",
+			mcp.WithDescription(t("TOOL_LIST_TAGS_DESCRIPTION", "List tags in a GitHub repository")),
```

Comment:
```suggestion
			mcp.WithDescription(t("TOOL_LIST_TAGS_DESCRIPTION", "List git tags in a GitHub repository")),
```

I think it might be better to be explicit here that they are git tags to avoid any ambiguity.